dev-python/nuitka/metadata.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<maintainer type="person">
+	<name>Oz Tiram</name>
```

Comment:
I'm sorry if somebody asked this already but is 'Oz' your real name?

Reply:
Thank you. It made me think of 'Wizard of Oz', that's why I asked. Hope I didn't offend you.

Reply:
I am not offended. It's often that people in Germany (where I live) ask about my name.

Reply:
I am not offended. It's often that people in Germany (where I live) ask about my name.

Reply:
I am not offended. It's often that people in Germany (where I live) ask about my name.

Reply:
I am not offended. It's often that people in Germany (where I live) ask about my name.

Reply:
I was not offended. It's quite common for people in Germany (where I live) to ask...